Actions/CheckForUpdates/CheckForUpdates.ps1

```diff
@@ -204,6 +204,14 @@ if ($update -ne 'Y') {
 else {
     # $update set, update the files
     try {
+        if (!$directCommit) {
+            $env:GH_TOKEN = $token
+            $existingPullRequest = (gh api --paginate "/repos/$env:GITHUB_REPOSITORY/pulls?base=$updateBranch" -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" | ConvertFrom-Json) | Where-Object { $_.title -eq "[$updateBranch] Update AL-Go System Files - $templateSha" } | Select-Object -First 1
+            if ($existingPullRequest) {
+                OutputWarning "Pull request already exists for [$updateBranch] Update AL-Go System Files - $($templateSha): $($existingPullRequest.html_url)."
```

Comment:
I see the constructed title is used in multiple places. Consider extracting it to a variable to make sure it's the same everywhere.